src/main/java/com/urlparser/entity/StringLinksFromUrl.java

```diff
@@ -0,0 +1,46 @@
+package com.urlparser.entity;
+
+import com.urlparser.model.Links;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class StringLinksFromUrl implements Links<String>{
```

Comment:
@undsett lets simplify it to `StringLinks`, constructor already says from which source(`String url`)
and for second impl relevant also